packages/verification-vue-component/src/components/Verification/items/cards/BaseCard.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-    <div class="base-card">
+    <div class="base-card" :class="cardClass">
```

Comment:
This is not necessary, if you add classes to a custom component they will be automatically added to the classes of the root element: https://vuejs.org/v2/guide/class-and-style.html#With-Components

Reply:
Ah yes, nice...changed it